Serializer/BaseSerializerHandler.php

```diff
@@ -27,6 +27,11 @@
     protected $manager;
 
     /**
+     * @var string[]
+     */
+    protected static $formats = array('json', 'xml', 'yml');
```

Comment:
It would be great to make this configurable, so

1. this should be `null`
2. if it is still `null` when `getSubscribingMethods` is called, it should be initialized to this array, and a deprecation warning should be triggered
3. To avoid the deprecation warning and prepare for the future, `SonataCoreExtension` should set it to the configured value
4. In the configuration, the value should default to `array('json', 'xml', 'yml')`